sympy/simplify/cse_diff.py

```diff
@@ -0,0 +1,188 @@
+"""Module for differentiation using CSE."""
+
+from sympy import cse, Matrix, SparseMatrix, Derivative, MatrixBase
+from collections import Counter
+
+
+def postprocess(repl, reduced):
```

Comment:
```suggestion
def _postprocess(repl, reduced):
```

Let's make all these private.